plugins/woocommerce/src/RestApi/Routes/V4/Settings/Products/Schema/ProductSettingsSchema.php

```
@@ -0,0 +1,292 @@
+
+		return $field;
+	}
+
+	/**
+	 * Get options for specific field types.
+	 *
+	 * @param string $field_type Field type.
+	 * @param string $field_id Field ID.
+	 * @return array Field options.
+	 */
+	private function get_field_options( string $field_type, string $field_id ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$field_type'. (undefined)

(UnusedFormalParameter)